package.json

```diff
@@ -7,7 +7,7 @@
   },
   "license": "AGPL-3.0-only",
   "author": "Internet Archive",
-  "version": "1.3.0",
+  "version": "1.3.1-alpha.0",
```

Comment:
Just a note to correct the version here before merging.

Reply:
Did, thanks for the note!